octodns/provider/route53.py

```diff
@@ -465,6 +750,75 @@ def _load_records(self, zone_id):
 
         return self._r53_rrsets[zone_id]
 
+    def _data_for_dynamic(self, name, _type, rrsets):
```

Comment:
This converts a bunch of "dynamic" rrsets into corresponding Record data, it's used by populate.